frontend/packages/console-dynamic-plugin-sdk/src/webpack/ConsoleRemotePlugin.ts

```diff
@@ -52,6 +52,13 @@ export class ConsoleRemotePlugin {
       compiler.options.output.enabledLibraryTypes.push(remoteEntryLibraryType);
     }
 
+    compiler.options.plugins.push(
```

Comment:
Please add the following comment above this line:

```
// Ignore require calls for modules exposed as plugin API
```

Reply:
commend added